components/AppBar/src/private/MDCAppBarViewController+ios.h

```diff
@@ -0,0 +1,28 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#if TARGET_OS_IOS
+
+#import "MDCAppBarViewController.h"
+
+/**
+  iOS specific implementation of controller of MDCAppBarViewController
+  */
+@interface MDCAppBarViewController (iOS)
```

Comment:
nit: The category should precisely be reflected in the filename, so here and below:
MDCAppBarViewController+ios.h
-TO-
MDCAppBarViewController+iOS.h  (Note capitalization of iOS)